src/DIRAC/AccountingSystem/scripts/dirac_accounting_decode_fileid.py

```diff
@@ -24,16 +18,15 @@
 from six.moves.urllib import parse as urlparse
 
 from DIRAC import gLogger
-from DIRAC.Core.Base import Script
 from DIRAC.Core.Utilities.DIRACScript import DIRACScript
 
 
 @DIRACScript()
 def main():
   from DIRAC.Core.Utilities.Plotting.FileCoding import extractRequestFromFileId
-  Script.parseCommandLine()
+  DIRACScript.registerArgument(["URL: encoded URL of a DIRAC Accounting plot"])
```

Comment:
In the examples above you always proposed to do

```
from DIRAC.Core.Utilities.DIRACScript import DIRACScript as Script
```

why not using the same style here (everywhere)?

Reply:
I tried to avoid unnecessary changes, so as not to make the big PR even bigger, but I can apply this style everywhere, I'll fix

Reply:
fixed